packages/docs/src/locales/en/en.json

```diff
@@ -3578,7 +3578,8 @@
           "Slots allow you to control the display of data and their views in different parts of the table. The examples below demonstrate some of these slots and what you can do with each of them.",
           "**Using custom slots**. Custom slots (*username*, *street* and *company name*) display certain data.",
           "**Showing header, body and footer's prepend and append static rows**. For added (prepended and appended) static rows an additional `className` with a background color is set.",
-          "**Using `colgroup` slot to set specific attributes to columns**. For last column an additional `className` with a background color is set."
+          "**Using `colgroup` slot to set specific attributes to columns**. For last column an additional `className` with a background color is set.",
+          "Note. The `columns` property is required while using slots. Slot names that not in `columns` will be omitted."
```

Comment:
"Slot names that are not in `columns` will be omitted."

Reply:
Fixed.